docs/docs/dcap.md

```diff
@@ -45,20 +30,43 @@ them if necessary).
 
     The bundled DCAP Azure client plugin works with all current generation Azure confidential VMs, as well as existing
     CPUs. However, if Azure extends their offerings to include new platforms, or if you use a new generation system, 
-    then the bundled Azure client plugin may throw an error when attempting to load your enclave. In this case,
+    then the bundled Azure client plugin might throw an error when attempting to load your enclave. In this case,
     [follow the instructions below](#azure-client-plugin) to build/install an updated version of the Azure client 
     plugin that supports the platform.
 
-You may want to set the Azure DCAP client logging level to FATAL as the default setting is quite verbose:
+You might want to set the Azure DCAP client logging level to FATAL as the default setting is quite verbose:
+
 ```sh
 export AZDCAP_DEBUG_LOG_LEVEL=FATAL
 ```
 
-If you are using the bundled version, you can skip the rest of this section. Otherwise, read on for instructions on how to manually configure the DCAP plugin.
+The Azure caches are open to the public internet and can be used from anywhere. Azure Confidential Computing 
+instances like DC4s_v3 come pre-provisioned for DCAP. As Conclave comes with the necessary libraries bundled, you 
+don't need to do any further setup.
+
+### DCAP Plugin
```

Comment:
This heading is wrong. I think it should be `Provisioning Certificate Caching Service (PCCS)` or something similar.

Reply:
I guess what I'm asking is what information is this section trying to convey? How to use PCCS, or something else?